shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Serialization.hs

```diff
@@ -137,3 +156,85 @@ rationalFromCBOR = do
     case ints of
       n:d:[] -> pure $ n % d
       _ -> cborError $ DecoderErrorSizeMismatch "rational" 2 numInts
+
+-------------------------------------------------------------------------
```

Comment:
We have an `AnnotatedDecoder` in `cardano-base`. It's now not used anywhere. I think it would be better to move this code there. If there's a good reason to not do that, then we should at least ensure that the code in `cardano-base` is deleted.

Reply:
This was a quicker stopgap. It should go into cardano-base.

The existing `fromCBORAnnotated` in cardano-base *is* used, so we'd need to resolve the name conflict one way or the other.